### ISSUE TITLE
Install script unification

### ISSUE THREAD

**Issue by sinban04** [SKKU-ESLAB/ANT#38]
In our project, there is target/ directory and install-deps script for each target.
(install-deps-tegraTX1.sh, install-deps-raspberry-pi2_3.sh)
In many open source projects, they use one script file and use argument to support several target platforms.
Therefore, i propose make the install-deps script one using argument with the target name.

Our install.sh script is already done with the argument of target board name.
(./install.sh --target=raspberry_pi2_3)
Which do you think it's better? unify the install-deps.sh script or leave it as it is now?

**Comment by LeeHayun** [comment]
Actually, install-deps-tegraTX1.sh and ...-raspberry-pi2_3.sh is very similar.
If you unify install-deps-*.sh, where is this file located?

**Comment by RedCarrottt** [comment]
Unifying ```install-deps-*.sh``` files is a good idea. As @LeeHayun said, they behavior in very similar manner.

**Comment by sinban04** [comment]
I expect that the install-deps.sh should be located in tools/ dir.
Or it's also okay in script/ dir.

**Comment by RedCarrottt** [comment]
@sinban04 
For the integrity, moving all the scripts in ```scripts``` directory (including ```install.sh```) to ```tools``` would be good.

Can you integrate ```tools/install-deps-rpi.sh``` for Travis CI docker with ```target/raspberry-pi2_3/install-deps-raspberry-pi2_3.sh``` for real device?

**Comment by sinban04** [comment]
I agree all the script files should be located tools/ dir.
The integration of two install-deps script (raspberry-pi2_3 and tegraTX1) should be done first,
and travis can use that script as it is.

**Comment by RedCarrottt** [comment]
I unified install script in PR #127 to use cmake and kconfig.
Therefore I close this issue.
